src/cli/cli.cpp

```diff
@@ -2646,7 +2682,9 @@ template <> otError Interpreter::Process<Cmd("csl")>(Arg aArgs[])
      * @endcode
      * @cparam csl channel @ca{channel}
      * @par api_copy
-     * #otLinkSetCslChannel
+     * @par
+     * `OPENTHREAD_CONFIG_MAC_CSL_RECEIVER_ENABLE` is required.
+     * @sa otLinkSetCslChannel
```

Comment:
Can you switch these back to what you had, with the api_copy?  This one and the next two.  They were fine.

Only the ones without api_copy tags needed to be changed.